### PR TITLE
[docs] Build release notes to documentation

### PR DESCRIPTION
**Pull request by bmorelli25**
For https://github.com/elastic/apm-agent-java/issues/943.

This PR adds release notes to the documentation for the Java Agent. Additional information and discussion is in elastic/apm#132.

The following PRs must be merged before this PR builds correctly:
- [x] https://github.com/elastic/apm-agent-java/pull/944
- [x] https://github.com/elastic/docs/pull/1458

## Landing page:
Screen Shot 2019-11-25 at 3 25 48 PM

## 1.x release notes:
Screen Shot 2019-11-25 at 3 25 59 PM

### CODE REVIEW

**Comment by felixbarny** [comment]
I have to confess something: the `CHANGELOG.md` is not really in sync with the [releases page](https://github.com/elastic/apm-agent-java/releases) 🙈 

It seems you have based these changes on the `CHANGELOG.md`. Is that right?

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+////
+[[release-notes-x.x.x]]
+==== x.x.x - YYYY/MM/DD
+
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+* Cool new feature: {pull}2526[#2526]
```
Comment (felixbarny) [inline CHANGELOG.asciidoc]: uh, nice!

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -0,0 +1,490 @@
+ifdef::env-github[]
+NOTE: Release notes are best read in our documentation at
+https://www.elastic.co/guide/en/apm/agent/ruby/current/release-notes.html[elastic.co]
+endif::[]
+
+////
+[[release-notes-x.x.x]]
+==== x.x.x - YYYY/MM/DD
```
Comment (felixbarny) [inline CHANGELOG.asciidoc]: should we back-fill the release dates?

**Comment by bmorelli25** [comment]
> I have to confess something: the CHANGELOG.md is not really in sync with the releases page 🙈
> It seems you have based these changes on the CHANGELOG.md. Is that right?

Ahhh, you're right, there are differences. I took a quick glance at the releases and it looked the same 🤦‍♂. I'll update to reflect the information in releases.

File: CHANGELOG.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply CHANGELOG.asciidoc]: Will do

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply CHANGELOG.asciidoc]: Easy links (once you get used to it)

**Comment by bmorelli25** [comment]
Added release dates and missing information from [releases page](https://github.com/elastic/apm-agent-java/releases).

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**